CHANGES.md

```diff
@@ -44,10 +44,11 @@ Unreleased
 * [#254](https://github.com/serokell/xrefcheck/pull/254)
   + Now the `dump-config` command does not overwrite a file unless explicitly told with a
     `--force` flag. Also, a `--stdout` flag allows to print the config to stdout instead.
-    the configured Markdown flavour.
```

Comment:
That's a good change :+1: 

And I would like it to be placed to a different commit :smile_cat:

Reply:
And the addition below should probably belong to the main commit )

Also, I would add a description to that commit you extracted after all. At a first glance, it's not clear why exactly you delete this line, maybe it was meant to contain some important information and should be extended, not removed. If in the commit description you mention that this was an accidental copy-paste, that would make the reasoning much clearer for an arbitrary git history reader.

Reply:
Very good, thank you :+1: